gravityflow/gravityflowparentchild
# Add Acceptance Tests for Parent Child Extension

## Description
Adds Acceptance Tests following acceptance test pattern as the other add-ons (Codeception, Selenium, Docker, etc.)
Includes tests to check and create parent-child entry relations and entries.

## Testing instructions
Run docker command to check all the acceptance tests.
`docker-compose run --rm codeception run -vvv --html`

## Automated Test Enhancements
Add tests `0001ParentChildEntryCept.php` and `0002ParentEntryIDCept.php`.

## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
